ang/crmMailing/services.js

```diff
@@ -60,18 +60,15 @@
     window.tpls = tpls;
     var lastModifiedTpl = null;
     return {
+      // Get a template
+      // @param id MessageTemplate id  (per APIv3)
       // @return Promise MessageTemplate (per APIv3)
       get: function get(id) {
-        id = '' + id; // parseInt(id);
-        var dfr = $q.defer();
-        var tpl = _.where(tpls, {id: id});
-        if (id && tpl && tpl[0]) {
-          dfr.resolve(tpl[0]);
-        }
-        else {
-          dfr.reject(id);
-        }
-        return dfr.promise;
+        return crmApi('MessageTemplate', 'get', {
```

Comment:
if you change "get" to "getsingle" then you can revert the changes to `ang/crmMailing/MsgTemplateCtrl.js` because it won't be buried in an array. Also you can then drop the "sequential" param.